src/components/Cta.astro

```diff
@@ -6,7 +6,7 @@ interface Action {
 export interface Props {
   title: string;
   subtitle?: string | null;
-  action: Action;
+  action: Action | null;
```

Comment:
Making it optional is usually a bit cleaner. Then you don't need to set the property when using the component instead of explicitly setting it to `null`.